p2p-specs/p2p-interface.md

```diff
@@ -400,7 +414,6 @@ The response MUST consist of a single `response_chunk`.
 Clients SHOULD immediately disconnect from one another following the handshake above under the following conditions:
 
 1. If the supported_mempools does not match with the client's own list of supported mempools.
```

Comment:
I know that this is not changed in this PR, but maybe it's better to be more specific? Like
1. If the supported_mempools an client's own list of supported mempools are disjoint.